modules/tinymce/src/core/main/ts/api/dom/DomQuery.ts

```diff
@@ -11,6 +11,8 @@ import Sizzle from './Sizzle';
 
 /**
  * This class mimics most of the jQuery API:
+ * <br>
+ * <em>`tinymce.dom.DomQuery` has been deprecated and marked for removal in TinyMCE 6.0.</em>
```

Comment:
```suggestion
 * <em><code>DomQuery</code> has been deprecated and marked for removal in TinyMCE 6.0.</em>
```
This can be confusing, as that's not how it's constructed at all times. As such we should just use the class name.